vtkVmtk/IO/vtkvmtkFDNEUTReader.cxx

```diff
@@ -52,6 +52,7 @@ vtkvmtkFDNEUTReader::~vtkvmtkFDNEUTReader()
     }
 }
 
+#if (VTK_MAJOR_VERSION >= 9 && VTK_MINOR_VERSION >= 1)
```

Comment:
This check will fail for VTK 10.0. Use VTK_VERSION_CHECK macro. Unfortunately, it is not available in VTK8, so it has to be added to VMTK if we really want to support VTK8.

Reply:
Thank you for the feedback @lassoan. I ran into this issue testing something else.
I don't think it is necessary (at least for me), but wanted to put it out there to get feedback.

What is the way forward to deprecating the use of vtk8 with vmtk? 
What is the path forward to supporting vtk10?

Reply:
Actually, we could merge this as is (knowing that it will not work with VTK10), because after we merge this, we can create a 1.4 branch (that will be compatible with VTK8) and then drop VTK8 support from the master branch (and remove this VTK10-incompatible fix).